test/backtrace.jl

```diff
@@ -4,68 +4,15 @@ bt = backtrace()
 have_backtrace = false
 for l in bt
     lkup = ccall(:jl_lookup_code_address, Any, (Ptr{Void},), l)
-    if lkup[1] == :backtrace
-        @test lkup[7] == false # fromC
+    if lkup[1][1] == :backtrace
+        @test lkup[1][5] == false # fromC
         have_backtrace = true
         break
     end
 end
 
 @test have_backtrace
 
-# Test location information for inlined code (ref issues #1334 #12544)
```

Comment:
if the results of these all changed, can we change them to test the affirmative of what they should be now, instead of deleting them altogether?